src/context/MessagingContext.js

```diff
@@ -112,8 +112,8 @@ export const MessagingProvider = (props) => {
     setIsLoading(false);
   };
 
-  const loadAuthors = async () => {
-    const res = await api.getAuthors();
+  const loadAuthors = async (organizationId) => {
+    const res = await api.getAuthors(organizationId);
 
     if (res.authors) {
       let result = res.authors.filter(
```

Comment:
The res doesn't have an authors property.  It has a growerAccounts property instead:

`const loadAuthors = async (organizationId) => {
    const res = await api.getAuthors(organizationId);

    if (res.growerAccounts) {
      let result = res.growerAccounts.filter(
        (author) => author.handle !== user.userName
      );
      setAuthors(result);
    }
  };`
  
  
  Otherwise, Nice Work! @ZavenArra

Reply:
ah got it